spec/acceptance/management_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper_acceptance'
+
+describe 'rancher::management' do
+  context 'puppet apply' do
+    it 'is expected to apply and be idempotent' do
+      pp = <<-EOS
+class { 'rancher::management':
+  rancher_manager_port => '8080',
+}
+EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      apply_manifest(pp, :catch_changes => true)
+    end
+
+    describe docker_container('rancher/server') do
+      it { is_expected.to be_running }
+    end
+
+    context 'is expected to be running on the default port' do
+      before :each do
+        sleep 5 # Give it time to start.
+      end
+
+      describe command('curl 0.0.0.0:8080') do
+        its(:stdout) { is_expected.to match /self.*0.0.0.0:8080/ }
```

Comment:
Yes I agree.  However it's stdout isn't including that string.  I wasn't sure if it's because it's broken but googling suggested that the only expectation should be for valid JSON.  In any case here's the output from curl:

```json
{
  "type": "collection",
  "resourceType": "apiVersion",
  "links": {
    "self": "http://0.0.0.0:8080/",
    "latest": "http://0.0.0.0:8080/v1"
  },
  "createTypes": {},
  "actions": {},
  "data": [
    {
      "id": "v1",
      "type": "apiVersion",
      "links": {
        "self": "http://0.0.0.0:8080/v1"
      },
      "actions": {}
    }
  ],
  "sortLinks": {},
  "pagination": null,
  "sort": null,
  "filters": {},
  "createDefaults": {}
}
```